torch/csrc/jit/frontend/schema_emitter.cpp

```diff
@@ -0,0 +1,145 @@
+#include <torch/csrc/jit/frontend/schema_matching.h>
```

Comment:
Nit:

I think this `schema_emitter.cpp` meant to match with `ir_emitter_utils.h`. It is customary to include its corresponding header file as first `include`. Also, I think it is better to change this file name to match `ir_emitter_utils.h`.